python/utilities/ev_whitelist/create_ev_whitelist_zip.py

```diff
@@ -21,13 +21,13 @@
 TRUNCATED_HASH_LENGTH = 8
 GOLOMB_M_PARAMETER = 2 ** 47
 NUM_FETCHING_PROCESSES = 40
-LOGS_LIST = [
-    "https://ct.googleapis.com/pilot",
-    "https://ct.googleapis.com/aviator",
-    "https://ct1.digicert-ct.com/log",
-    "https://log.certly.io",
-    "https://ct.izenpe.com",
-]
+LOGS_LIST = {
+    "https://ct.googleapis.com/pilot":6082932,
```

Comment:
(1) I manually found them from looking at logs of my January 1st run.
(2) They would never get refreshed - the whitelist was generated once to include all EV certs logged prior to January 1st. The goal of regenerating it is reducing its size (hence memory footprint on Chrome clients).